test/CMakeLists.txt

```diff
@@ -60,57 +52,107 @@ else()
     set(CRTM_TEST_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
 endif()
 
-set( CRTM_COEFFS_BRANCH "3.0.0" )
+set( CRTM_COEFFS_BRANCH "3.0.1" )
 
 # If local path to testfiles is defined don't download
-#if( DEFINED LOCAL_PATH_JEDI_TESTFILES )
 IF(EXISTS  ${CMAKE_SOURCE_DIR}/fix)
   set( CRTM_COEFFS_PATH ${CMAKE_SOURCE_DIR}/fix )
-  message(STATUS "use LOCAL_PATH_JEDI_TESTFILES: ${CRTM_COEFFS_PATH}")
-  message("Using existing local fix directory instead of downloading.")
-# Download CRTM coefficients
+  message(STATUS "Use CRTM_COEFFS_PATH: ${CRTM_COEFFS_PATH}")
```

Comment:
Not as part of this PR, but afterwards you could steal the new download logic from the ioda/ufo/... PRs and make this shorter and less complicated by using standard cmake utilities rather than in-house solutions.

Reply:
thanks, will look at that for the next iteration of updates shortly after this one.